src/bench.ts

```diff
@@ -207,6 +207,19 @@ export class Bench extends EventTarget {
     return values
   }
 
+  runSync (): Task[] {
+    if (this.opts.warmup) {
```

Comment:
A warning about not supporting concurrent benchmark (`concurrency` set to a non `null` value) should be issued.